module/token/helper.js

```diff
@@ -0,0 +1,55 @@
+
+/**
+ * name : token/helper.js
+ * author : Rakesh Kumar
+ * created-date : 21-july-2020
+ * Description : Related to user token validation
+ * to cloud service.
+ */
+
+// Dependencies
+const apiInterceptor = require(PROJECT_ROOT_DIRECTORY + "/generics/middleware/apiInterceptor");
+const shikshalokam = require(PROJECT_ROOT_DIRECTORY + "/generics/helpers/shikshalokam");
+
+
+/**
+* learning resource related information be here.
+* @method
+* @class  TokenHelper
+*/
+
+module.exports = class TokenHelper {
+
+/**
+* To validate the user token
+* @method
+* @name  verify
+* @param {String} token - user access token.
+* @returns {json} Response consists of user token details
+*/
+static verify(token) {
+    return new Promise(async (resolve, reject) => {
+       try {
+      
+       apiInterceptor.validateToken(token, async function (err, tokenData) {
```

Comment:
@rakeshSgr - Let the API of verification token response from the controller be like 
`{
    "message": "Token verified",
    "status": 200,
    "result": {
        "isValid": true/false,
         "userInformation" : {
 // If isValid token return the jwt info
          }
}
`